lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -9,7 +9,11 @@ def initialize(engine_config:, directory:)
 
         def run
           files.map do |file|
-            process_file(file)
+            begin
+              process_file(file)
+            rescue => e
+              $stderr.puts e
```

Comment:
I think we should stick with the full backtrace since we don't know how much we need to display for the information to be useful?